themes/default/content/blog/announcing-aws-native/index.md

```diff
@@ -1,8 +1,9 @@
 ---
-title: "Announcing the Pulumi AWS Native Provider"
+title: "Announcing the Pulumi AWS Native Provider, powered by the AWS Cloud Control API"
```

Comment:
```suggestion
title: "Announcing the Pulumi AWS Native Provider, Powered by the AWS Cloud Control API"
```

Reply:
suggest applying title case

Reply:
We should write down a consistent policy here at some point. I'm personally not a fan of title case, it feels very shout-y.  But, I'll take it here for now.